project-keeper-cli/pom.xml

```diff
@@ -98,17 +98,10 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <configuration>
-                    <!-- Javadoc causes error "error: package java.util.logging is not visible
-                        import java.util.logging.*;
-                        (package java.util.logging is declared in module java.logging, but module jakarta.json does not read it)". -->
-                    <additionalOptions>
-                        <option>--add-modules</option>
-                        <option>java.logging</option>
-                        <option>--add-reads</option>
-                        <option>jakarta.json=java.logging</option>
```

Comment:
I fixed this using `<option>org.glassfish.jakarta.json=java.sql</option>` but I guess this also works.